patches/minecraft/net/minecraft/fluid/Fluid.java.patch

```diff
@@ -24,29 +24,34 @@
 +    * Creates the fluid attributes object, which will contain all the extended values for the fluid that aren't part of the vanilla system.
 +    * Do not call this from outside. To retrieve the values use {@link Fluid#getAttributes()}
 +    */
-+   protected net.minecraftforge.fluids.FluidAttributes createAttributes(Fluid fluid)
++   protected net.minecraftforge.fluids.FluidAttributes createAttributes()
 +   {
```

Comment:
This would be better off moving to a ForgeHooks method, just due to the shear size, and uglyness that is due to being in a patch.

Reply:
Done.